src/openforms/contrib/digid_eherkenning/signals.py

```diff
@@ -0,0 +1,19 @@
+from django.db.models.signals import post_save
+from django.dispatch import receiver
+
+from digid_eherkenning.models.digid import DigidConfiguration
+from digid_eherkenning.models.eherkenning import EherkenningConfiguration
+
+from openforms.contrib.digid_eherkenning.utils import (
```

Comment:
```suggestion
from .utils import (
```
within a django app, please use relative imports. Use absolute imports if you're importing stuff from another app (this helps reviewing if you're importing stuff you shouldn't be importing ;) )